scripts/check_golang_version.sh

```diff
@@ -16,7 +16,7 @@ set -eo pipefail
 # fail the first check (GO_MOD_SUPPORT), and this isn't necessary for travis builds.
 #
 # https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
-if $CI && $TRAVIS
+if [[ ! -z $CI ]] && [[ ! -z $TRAVIS ]]
```

Comment:
Since travis sets these env vars as booleans, the check works without the brackets.  What you changed *should* still work, we'll find out soon enough :)

Reply:
The problem is these are not set when invoked from Makefile locally and the condition always met